docker-compose.yml

```diff
@@ -0,0 +1,22 @@
+version: '3.3'
+
+services:
+    api:
+        image: stocksimulator-api:1.0
+        build: ./api
+        ports:
+            - '5000:5000'
+
+    database:
+        image: postgres:11
+        restart: always
+        environment:
+            POSTGERS_USER: user
+            POSTGRES_PASSWORD: password
```

Comment:
so secure. much hacks. wow.

Reply:
Let's consider using `dotenv` or something similar and not commit our passwords to GitHub

I have a feeling that the professor _might_ be on our case about it. We can submit a dotenv via the mycourses dropbox :)

Reply:
Feature, not a bug, gonna refactor once we have functioning containers that you guys can build in